src/components/StorageOverview/Utilization/Utilization.js

```diff
@@ -55,13 +61,26 @@ export const Utilization = ({
   if (latencyStats) {
     latencyStatsMax = Math.max(0, ...latencyStats);
   }
+  const { timestamps } = throughputData;
 
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Performance</DashboardCardTitle>
+        <DashboardCardTitle>Utilization</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
+        <Row>
+          <Col className="kubevirt-utilization__time-duration time-duration--left">Time </Col>
```

Comment:
className should look like `className="kubevirt-utilization__time-duration kubevirt-utilization__time-duration--left"`. Same applies for `right`. It will conform to BEM and there will be no need to disable it in `scss`

Reply:
I tried that and it still gave the `Invalid component selector` error and hence had to disable it.

Reply:
does not happen here..

Reply:
`Invalid component selector                                                                          plugin/selector-bem-pattern
          ".kubevirt-utilization__time-duration.kubevirt-utilization__time-duration--left"`

Error ^

Reply:
@AyushAmbastha see suggested change which is passing the validation